osf/models/user.py

```diff
@@ -680,14 +682,45 @@ def merge_user(self, user):
             node.save()
 
         # - projects where the user was the creator
-        user.created.filter(is_bookmark_collection=False).update(creator=self)
+        user.created.filter(is_bookmark_collection=False).exclude(type='osf.quickfiles').update(creator=self)
 
         # - file that the user has checked_out, import done here to prevent import error
         from osf.models import BaseFileNode
         for file_node in BaseFileNode.files_checked_out(user=user):
             file_node.checkout = self
             file_node.save()
 
+        # - move files in the merged user's quickfiles node, checking for name conflicts
+        from osf.models import QuickFiles
+        primary_quickfiles = QuickFiles.objects.get(creator=self)
+        merging_user_quickfiles = QuickFiles.objects.get(creator=user)
+
+        files_in_merging_user_quickfiles = BaseFileNode.objects.filter(node=merging_user_quickfiles, type='osf.osfstoragefile')
+        for merging_user_file in files_in_merging_user_quickfiles:
```

Comment:
Could this fall prey to the queryset modifying issue that was present in the migration? You're filtering on `node`, but modifying `node`.